bootstrap/helpers/databases.php

```
@@ -75,6 +76,39 @@ function create_standalone_redis($environment_id, $destination_uuid, ?array $oth
+    $database->save();
+
+    EnvironmentVariable::create([
+        'key' => 'VALKEY_PASSWORD',
+        'value' => $valkey_password,
+        'resourceable_type' => StandaloneValkey::class,
+        'resourceable_id' => $database->id,
+        'is_shared' => false,
+    ]);
+
+    return $database;
+}
```

PHPMD (2.15.0)

The parameter $environment_id is not named in camelCase. (undefined)

(CamelCaseParameterName)